src/ui/lib/components/header.svelte

```diff
@@ -37,7 +38,8 @@
 		display: flex;
 		align-items: center;
 		background-color: #0d133d;
-		color: #ffffffdd;
+		// palette.shared.text.secondary.onDark
+		color: rgba(255, 255, 255, 0.7);
```

Comment:
I believe this can just be var(--mdc-theme-text-secondary-on-dark)

Reply:
you right, would be good to add the the docs on theme usage on how the palette object can be translated to css vars